package.json

```diff
@@ -55,15 +35,29 @@
     "watch:css": "node-sass-chokidar ./src/styles/all.scss ./src/styles/all.css --include-path ./node_modules src/styles -o src/styles --watch --recursive"
   },
   "devDependencies": {
+    "babel-eslint": "^10.0.1",
+    "coveralls": "^3.0.2",
+    "deep-freeze": "0.0.1",
+    "enzyme": "^3.7.0",
+    "enzyme-adapter-react-16": "^1.6.0",
+    "enzyme-to-json": "^3.3.4",
     "eslint": "^5.8.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-json": "^1.2.1",
     "eslint-plugin-jsx-a11y": "^6.1.2",
     "eslint-plugin-react": "^7.11.1",
+    "expect": "^23.6.0",
     "jest-junit": "^4.0.0",
+    "moxios": "^0.4.0",
     "node-sass-chokidar": "^1.3.3",
-    "react-addons-test-utils": "^15.6.2"
+    "react-addons-test-utils": "^15.6.2",
+    "react-mock-router": "^1.0.15",
+    "redux-devtools-extension": "^2.13.5",
+    "redux-logger": "^3.0.6",
+    "redux-mock-store": "^1.5.3",
+    "redux-test-utils": "^0.3.0",
+    "sinon": "^7.1.1"
```

Comment:
Is this being used?

Reply:
Yes @dessHub, `sinon` is being used in `src/tests/Dictionary/DictionaryConatiner.test.js`